movieApp/asset/app.js

```diff
@@ -0,0 +1,50 @@
+//const API_KEY=daf7b11fd7dfe1a364da7a86d79ba727;
+const IMG_URL="https://image.tmdb.org/t/p/w500";
+const url="https://api.themoviedb.org/3/search/movie?api_key=daf7b11fd7dfe1a364da7a86d79ba727";
+
+//selecting search elements from DOM
+const inputElement=document.querySelector("#searchBox");
+const movieSearchable=document.querySelector("#movies-searchable");
+
+inputElement.onsearch=function(event){//searchFunction()
```

Comment:
try to not leave commented code in the code which is donna be reviewed and then merged to master. If you want to have a comment put it above the function (in case you want to describe/clarify how function works)